source/Octopus.Tentacle/Diagnostics/Metrics/EventRecord.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Octopus.Tentacle.Diagnostics.Metrics
```

Comment:
These concepts are quite specific to Kubernetes. Can these files be moved to a `Octopus.Tentacle.Kubernetes.Diagnostics` namespace instead?

Reply:
Done